docs/contributing/how-to-open-a-pull-request.md

```diff
@@ -31,30 +31,40 @@ The Gatsby core team uses a triaging process outlined in [Managing Pull Requests
 
 ## Opening PRs in Gatsby
 
-For any kind of change to files in the Gatsby repo, you can follow the below steps. Be sure to check out additional tips for contributing to various parts of the repo later in this doc, such as docs changes, blog posts, starters, or code improvements.
+For any kind of change to files in the Gatsby repo, you can follow the below steps. Be sure to check out additional tips for contributing to various parts of the repo later in this doc, such as docs changes, blog posts, starters, or code improvements and tests.
 
-Some PRs can be done completely from the [GitHub UI](https://help.github.com/en/articles/creating-a-pull-request). To test changes locally against the Gatsby [site and project files](https://github.com/gatsbyjs/gatsby), you can fork the repo and install parts of it to run on your local machine.
+Some PRs can be done completely from the [GitHub UI](https://help.github.com/en/articles/creating-a-pull-request), such as edits to README files or docs.
+
+To test changes locally against the Gatsby [site and project files](https://github.com/gatsbyjs/gatsby), you can fork the repo and install parts of it to run on your local machine.
 
 - [Fork and clone the Gatsby repo](/contributing/setting-up-your-local-dev-environment/#gatsby-repo-install-instructions).
 - Install [yarn](https://yarnpkg.com/) to pull in dependencies and build the project.
 - Follow the instructions for the part of the project you want to change. (See specific sections below.)
 - [Create a branch in Git](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) to isolate your changes:
 
-  ```bash
+  ```shell
   git checkout -b some-change
   ```
 
-- Once you have changes in Git you want to push, [add them and create a commit](https://help.github.com/en/articles/adding-a-file-to-a-repository-using-the-command-line).
+- Once you have changes in Git you want to push, [add them and create a commit](https://help.github.com/en/articles/adding-a-file-to-a-repository-using-the-command-line). For information on how to structure your commits, check out the [Managing PRs](/contributing/managing-pull-requests/#commit-and-pr-title) doc.
   - Using a dot character `.` will add all untracked files in the current directory and subdirectories.
-  ```bash
+  ```shell
   git add .
   ```
   - Using a visual tool like [GitHub Desktop](https://desktop.github.com/) or [GitX](https://rowanj.github.io/gitx/) can help for choosing which files and lines to commit.
+- Committing code will run the automated linter using [Prettier](https://prettier.io). To run the linter manually, run an npm script in the project's base directory:
```

Comment:
```suggestion
- Committing code will run the automated [linter](/docs/glossary#linting) using [Prettier](https://prettier.io). To run the linter manually, run an npm script in the project's base directory:
```

Maybe add a reference in the glossary to linting? Likely not familiar for everyone.

Reply:
Oh dang, I was too quick on the merge! That would have been a good change. Perhaps a good follow-up PR from someone? I won't be able to get to it right away.